samples/requirements.txt

```diff
@@ -0,0 +1,5 @@
+pycuda
+tqdm
+torchvision
+onnx
+tensorrt
```

Comment:
could we just add this to the extras of setup.py? There would otherways to much sets of requirements.

Extras can be installed by naming them during pip install `pip install .[samples]` to install all dependencies of the "samples" extra in setup.py or `pip install .[samples-all]`/`pip install .[samples-pytorch]`

Reply:
I like that idea ! I'll check it out.